config/fcos/v1_5_exp/validate_test.go

```diff
@@ -231,3 +231,40 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateDisk(t *testing.T) {
+	tests := []struct {
+		in      Disk
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid disk
```

Comment:
We probably want some negative testing as well, try and think of some different permutations that cause different results that goes through this newly added code.

Reply:
added a negative part to the test but it seems to make the test fail altogether, not sure were i'm going wrong will look into it!